platform/darwin/resources/fr.lproj/Foundation.strings

```diff
@@ -299,8 +299,5 @@
 "OSM_SHORT_NAME" = "OSM";
 
 /* User-friendly error description */
-"REMOVE_SRC_FAIL_IN_USE_FMT" = "La source '%@' est en cours d'utilisation, impossible de la supprimer.";
-
-/* User-friendly error description */
-"REMOVE_SRC_FAIL_MISMATCH_FMT" = "L'identifiant '%1$@' ne correspond pas à l'identifiant de la source '%2$s'";
```

Comment:
I had to remove this translation, which was added in #13360, because the new source string takes only a single string placeholder, not two string placeholders.